snap/snapcraft.yaml

```diff
@@ -0,0 +1,231 @@
+name: yubioath-desktop
+adopt-info: yubioath-desktop
+icon: resources/icons/com.yubico.yubioath.svg
+base: core18
+grade: devel
+confinement: strict
+
+plugs:
+  ykman-config:
+    interface: personal-files
+    write:
+      - $HOME/.ykman
+apps:
+  pcscd:
+    command: usr/sbin/pcscd --foreground --auto-exit
+    daemon: simple
+    common-id: com.yubico.pcscd
+    plugs:
+      - hardware-observe
+      - network
+      - network-bind
+      - raw-usb
+
+  yubioath-desktop:
+    adapter: full
+    command: yubioath-desktop
+    common-id: com.yubico.yubioath
+    environment:
+      LD_LIBRARY_PATH: $SNAP/opt/qt514/lib/x86_64-linux-gnu/:$SNAP/opt/qt514/lib/:$LD_LIBRARY_PATH:$SNAP/usr/lib/
+      QT_QPA_PLATFORM: xcb
+      QT_BASE_DIR: $SNAP/opt/qt514/
+      QT_PLUGIN_PATH: $SNAP/opt/qt514/plugins/
+      QML2_IMPORT_PATH: $SNAP/opt/qt514/qml/
+    plugs:
+      - desktop
+      - desktop-legacy
+      - wayland
+      - unity7
+      - opengl
+      - x11
+      - raw-usb
+      - u2f-devices
+      - hidraw
+      - hardware-observe
+      - network
+      - network-bind
+      - network-observe
```

Comment:
Offline conversation: double check that network plugs are needed

Reply:
Dropped!